qbraid_qir/qiskit/elements.py

```
@@ -0,0 +1,189 @@
+    """Represents a Qiskit quantum circuit prepared for QIR conversion.
+
+    Attributes:
+        circuit: The original Qiskit QuantumCircuit.
+        name: The name of the module.
+        module: The PyQIR Module being built.
+        num_qubits: Number of qubits in the circuit.
+        num_clbits: Number of classical bits in the circuit.
+        reg_sizes: List of sizes for each classical register.
+    """
+
+    def __init__(
```

Ruff (0.15.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)